src/base/search/searchpluginmanager.cpp

```diff
@@ -446,9 +448,25 @@ void SearchPluginManager::applyProxySettings()
 
     case Net::ProxyType::SOCKS5:
         {
-            const QString proxyURL = proxyConfig.authEnabled
-                ? u"%1:%2@%3:%4"_s.arg(proxyConfig.username, proxyConfig.password, proxyConfig.ip, QString::number(proxyConfig.port))
-                : u"%1:%2"_s.arg(proxyConfig.ip, QString::number(proxyConfig.port));
+            const QString scheme = proxyConfig.hostnameLookupEnabled ? u"socks5h"_s : u"socks5"_s;
```

Comment:
Where do these URL schemes come from? Are they real or fake (qBittorrent private)?

Reply:
>Where do these URL schemes come from?

https://everything.curl.dev/usingcurl/proxies/socks.html

>Are they real or fake (qBittorrent private)?

Many of the http frameworks in python follows the notion from curl. I figured it would be best if qbt followed it too. Example: https://www.python-httpx.org/advanced/proxies/#socks